docs/specification.md

```diff
@@ -689,33 +684,6 @@ Parameter (in Michelson):
 - Fails with `NOT_ADMIN` if the sender is not the administrator.
 - Fails with `OUT_OF_BOUND_VOTING_PERIOD` if the voting period value is out of the bound set by the configuration
 
-### **set_quorum_threshold**
-
-```ocaml
-// Quorum threshold that a proposal needs to meet in order to be accepted,
-// expressed as a fraction of the total_supply of frozen tokens.
-// Invariant: numerator < denominator
-type quorum_threshold =
-  [@layout:comb]
-  { numerator : nat
-  ; denominator : nat
-  }
-
-Set_quorum_threshold of quorum_threshold
-```
-
-Parameter (in Michelson):
-```
-(pair %set_quorum_threshold (nat %numerator) (nat %denominator))
-```
-
-- Update the quorum threshold which proposals have to meet to not get rejected.
-- The Quorum threshold is calculated as the proportion of total votes (upvotes
-  and downvotes) over the frozen token's total supply.
-- This affects all ongoing and new proposals.
-- Fails with `NOT_ADMIN` if the sender is not the administrator.
-- Fails with `OUT_OF_BOUND_QUORUM_THRESHOLD` if the quorum threshold value is out of the bound set by the configuration or the numerator is not smaller than the denominator.
```

Comment:
Not that this entrypoint has been removed the `OUT_OF_BOUND_QUORUM_THRESHOLD` is no longer used.

We should remove it from the `Errors` section in this doc as well as removing the instances in `Ligo.BaseDAO.Types`

Reply:
Updated